packages/backend-function/API.md

```diff
@@ -26,11 +29,18 @@ export type FunctionProps = {
     memoryMB?: number;
     environment?: Record<string, string | BackendSecret>;
     runtime?: NodeVersion;
+    schedule?: TimeInterval | TimeInterval[];
 };
 
 // @public (undocumented)
 export type NodeVersion = 16 | 18 | 20;
 
+// @public (undocumented)
+export type Rate = `every ${number}m` | `every ${number}h` | `every day` | `every week` | `every month` | `every year`;

```

Comment:
Suggestion for alternative namings:

```ts
type TimeInterval = `every xx`;
type CronSchedule = `{string} ...`
type Schedule = TimeInterval | CronSchedule
```

Reply:
I was avoiding `type Schedule` because of the naming clash with the Schedule class from `aws-cdk-lib/aws-events`. Alternatively we can use `FunctionSchedule`?